geode-core/src/main/java/org/apache/geode/management/internal/cli/shell/GfshExecutionStrategy.java

```diff
@@ -254,14 +252,26 @@ private Result executeOnRemote(GfshParseResult parseResult) {
       CommandResult postExecResult =
           interceptor.postExecution(parseResult, commandResult, tempFile);
       if (postExecResult != null) {
-        if (Status.ERROR.equals(postExecResult.getStatus())) {
+        boolean errored = false;
+        if (postExecResult instanceof ResultModel) {
```

Comment:
would postExecResult ever be an instance of ResultModel? If so, line 252 would not be correct....

Reply:
Right - good catch. That check is actually redundant - it will always be a `CommandResult`. Leftovers from initial refactoring work.